packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/add-groups-claim-dynamic/index.md

```diff
@@ -16,11 +16,11 @@ You can use this function anywhere to get a list of Groups of which the current
 >
 >Example: `getFilteredGroups({"00gml2xHE3RYRx7cM0g3"}, "group.name", 40) )`
 >
->See the **Parameter Examples** section of [Use group functions for static group whitelists](/docs/guides/customize-tokens-static/static-allowlist/#use-group-functions-for-static-group-whitelists) for more information on the parameters used in this Group function.
+>See the **Parameter Examples** section of [Use group functions for static group allow lists](/docs/guides/customize-tokens-static/static-allowlist/#use-group-functions-for-static-group-allow-lists) for more information on the parameters used in this Group function.
```

Comment:
After this change, we might need to change the archor name.

Reply:
Anchors are auto-generated by VuePress based on the heading.